browser/brave_content_browser_client.cc

```diff
@@ -281,31 +291,18 @@ void BraveContentBrowserClient::MaybeHideReferrer(
   }
 
   Profile* profile = Profile::FromBrowserContext(browser_context);
-  const bool allow_referrers =
-      brave_shields::IsAllowContentSettingsForProfile(
-          profile,
-          document_url,
-          document_url,
-          CONTENT_SETTINGS_TYPE_PLUGINS,
-          brave_shields::kReferrers);
-  const bool shields_up =
-      brave_shields::IsAllowContentSettingsForProfile(
-          profile,
-          document_url,
-          GURL(),
-          CONTENT_SETTINGS_TYPE_PLUGINS,
-          brave_shields::kBraveShields);
+  const bool allow_referrers = brave_shields::IsAllowContentSettingsForProfile(
+      profile, document_url, document_url, CONTENT_SETTINGS_TYPE_PLUGINS,
+      brave_shields::kReferrers);
+  const bool shields_up = brave_shields::IsAllowContentSettingsForProfile(
+      profile, document_url, GURL(), CONTENT_SETTINGS_TYPE_PLUGINS,
+      brave_shields::kBraveShields);
 
   // Navigations get empty referrers (brave/brave-browser#3422).
   const GURL empty_referrer_url;
-  brave_shields::ShouldSetReferrer(allow_referrers,
-                                   shields_up,
-                                   referrer->url,
-                                   document_url,
-                                   request_url,
-                                   empty_referrer_url,
-                                   referrer->policy,
-                                   referrer);
+  brave_shields::ShouldSetReferrer(
```

Comment:
This is a result of `clang-format -i`. I think reverting this makes sense.

Reply:
why do you think so? Formatting looks correct

Reply:
I find the earlier code looks a little bit more readable. I can keep it as is if its okay to you and others.

Reply:
well, at least all formatting now conforms to CC. I'm for using clang-format as a baseline

Reply:
Ok cool, will leave it as is. Thanks @iefremov .